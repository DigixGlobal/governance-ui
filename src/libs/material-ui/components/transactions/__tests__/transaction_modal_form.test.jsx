/* eslint global-require:0 */

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TransactionModalForm from '../transaction_modal_form';

jest.mock('../../../helpers/web3/connect', () => require('../../../../test/__mocks__/connectMock'));

describe('<TransactionModalForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <TransactionModalForm
        formChange={jest.fn}
        formData={{}}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
