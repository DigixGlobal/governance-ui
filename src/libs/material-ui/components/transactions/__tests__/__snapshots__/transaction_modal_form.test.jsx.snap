// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionModalForm /> renders correctly 1`] = `
<div>
  <Advanced
    title="Advanced"
  >
    <FormGroup
      widths="equal"
    >
      <FormField
        formChange={[Function]}
        formData={Object {}}
        label="Gas"
        name="gas"
        placeholder="e.g. \`21000\`"
        type="number"
      />
      <FormField
        formChange={[Function]}
        formData={Object {}}
        label="Gas Price"
        name="gasPrice"
        placeholder="e.g. \`20000000000\`"
        type="number"
      />
      <FormField
        formChange={[Function]}
        formData={Object {}}
        label="Nonce"
        name="nonce"
        placeholder="Leave empty to auto-detect"
        type="number"
      />
    </FormGroup>
  </Advanced>
</div>
`;
